frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/general-tab.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Grid } from '@patternfly/react-core';
+import {
+  iGetRelevantTemplateSelectors,
+  iGetVmSettings,
+} from '../../selectors/immutable/vm-settings';
+import { VMSettingsField, VMWizardProps } from '../../types';
+import { FormFieldReviewMemoRow } from '../../form/form-field-review-row';
+import { FormFieldType } from '../../form/form-field';
+import { iGetCommonData, iGetLoadedCommonData } from '../../selectors/immutable/selectors';
+import { getOS } from '../../selectors/common';
+import { getFieldValue } from '../../selectors/vm-settings';
+import { iGetRelevantTemplate } from '../../../../selectors/immutable/template/combined';
+import { toShallowJS } from '../../../../utils/immutable';
+import { VMTemplateWrapper } from '../../../../k8s/wrapper/vm/vm-template-wrapper';
+import { CUSTOM_FLAVOR } from '../../../../constants/vm';
+import { iGetFieldValue } from '../../selectors/immutable/field';
+import { getField } from './utils';
+import { VMSettings } from '../../redux/initial-state/types';
+
+import './general-tab.scss';
+
+const GeneralReviewConnected: React.FC<GeneralReviewConnectedProps> = (props) => {
+  const { iVMSettings, iUserTemplates, iCommonTemplates, openshiftFlag, relevantOptions } = props;
+
+  const getFlavorValue = () => {
+    const flavor = iGetFieldValue(getField(VMSettingsField[VMSettingsField.FLAVOR], iVMSettings));
+    let cpuCores, memory;
+
+    if (flavor === CUSTOM_FLAVOR) {
+      cpuCores = iGetFieldValue(getField(VMSettingsField.CPU, iVMSettings));
+      memory = iGetFieldValue(getField(VMSettingsField.MEMORY, iVMSettings));
+    } else {
+      const template_ = toShallowJS(
+        iGetRelevantTemplate(iUserTemplates, iCommonTemplates, relevantOptions),
+      );
+      const template = new VMTemplateWrapper(template_, true).init().clearRuntimeMetadata();
+
+      cpuCores = template.getCPU()?.cores;
+      memory = template.getMemory();
+    }
+
+    return `${flavor}: ${cpuCores} CPU, ${memory}`;
+  };
+
+  const osName =
+    getOS({
+      osID: getFieldValue(iVMSettings.toJS() as VMSettings, VMSettingsField.OPERATING_SYSTEM),
```

Comment:
no need for toJS here

Reply:
It doesn't work properly if I remove it. `osName` ends up just being the ID and not the name. I'll look at it more if I have time. I'll leave it as it is for now since it's working.

Reply:
Done!